sdk/azidentity/client_certificate_credential_test.go

```diff
@@ -243,6 +244,10 @@ func TestClientCertificateCredential_InvalidCertLive(t *testing.T) {
 	if e.RawResponse == nil {
 		t.Fatal("expected a non-nil RawResponse")
 	}
+	expectedErrorMsg := "ClientCertificateCredential authentication failed"
+	if !strings.Contains(e.Error(), expectedErrorMsg) {
+		t.Fatalf("unexpected error message: expected error to contain %s", expectedErrorMsg)
+	}
```

Comment:
Could probably just simplify this to checking whether the credential name is in the message:
```suggestion
	if !strings.Contains(e.Error(), credNameCert) {
		t.Fatalf("error doesn't contain credential name")
	}
```

Reply:
Resolved in commit [00753ad](https://github.com/Azure/azure-sdk-for-go/pull/18779/commits/00753addd6bde38cd7375b0cb638512cc5838934)